lib/src/working_copy.rs

```diff
@@ -69,7 +69,10 @@ pub struct FileState {
 }
 
 impl FileState {
+    #[cfg_attr(windows, allow(unused_variables))]
     fn for_file(executable: bool, size: u64, metadata: &Metadata) -> Self {
+        #[cfg(windows)]
+        let executable = false;
```

Comment:
Maybe we can instead turn the `executable` flag to `()` on Windows?

```rust
#[cfg(windows)]
type FileExecutableFlag = ();
...
pub enum FileType {
    Normal { executable: FileExecutableFlag },
...
```

We'll need a few more `#[cfg(windows)]`, but it ensures that the executable flag is never compared on Windows.

Reply:
Yeah, I considered leaving out the flag completely, but making it a type alias as you suggest is a bit better. Done.